Source/ui_unix/mainwindow.cpp

```diff
@@ -278,29 +281,26 @@ void MainWindow::SetupSaveLoadStateSlots()
         QAction* saveaction = new QAction(this);
         saveaction->setText(QString("Save Slot %1 - %2").arg(i).arg(info));
         saveaction->setEnabled(enable);
-        saveaction->setProperty("stateSlot", i);
         ui->menuSave_States->addAction(saveaction);
 
         QAction* loadaction = new QAction(this);
         loadaction->setText(QString("Load Slot %1 - %2").arg(i).arg(info));
         loadaction->setEnabled(enable);
-        loadaction->setProperty("stateSlot", i);
         ui->menuLoad_States->addAction(loadaction);
 
         if (enable)
         {
-            connect(saveaction, SIGNAL(triggered()), this, SLOT(saveState()));
-            connect(loadaction, SIGNAL(triggered()), this, SLOT(loadState()));
+            connect(saveaction, &QAction::triggered, std::bind(&MainWindow::saveState, this, i));
+            connect(loadaction, &QAction::triggered, std::bind(&MainWindow::loadState, this, i));
         }
 
     }
 }
 
-void MainWindow::saveState()
+void MainWindow::saveState(int m_stateSlot)
```

Comment:
Prefixing names with `m_` is reserved for class member variables. This should be `stateSlot`.